src/cdk/rxjs/rx-operators.ts

```diff
@@ -89,6 +89,7 @@ export class StartWithBrand { private _; }
 export class DebounceTimeBrand { private _; }
 export class AuditTimeBrand { private _; }
 export class TakeUntilBrand { private _; }
```

Comment:
I'd be ok with having these properties be public. They're not a part of any API or generated JS anyway.

Reply:
Yeah I don't mind either. I just changed it. (doesn't really matter)

Reply:
The brand properties actually _must_ be `private`, otherwise they are structurally identical to anything and everything.